docs/proyectos-de-codigo-abierto.mdx

```diff
@@ -426,6 +426,20 @@ También permite que cualquier persona que posea el documento valide el registro
 
 ### Entornos de Desarrollo Local
 
+EOS Costa Rica ofrece una amplia variedad de entornos de desarrollo local para la comunidad de desarrolladores.
+
+<div class="table-wrapper" markdown="block" style={{overflow: 'scroll', height: '200px'}}>
+
+| Entornos de Desarrollo Local | Descripción | 
+| :----------: | :---------: | 
+| [EOS](https://github.com/eoscostarica/eos-local) | Red EOS basada en Docker para el desarrollo local. | 
+| [Telos](https://github.com/eoscostarica/telos-local)| Red Telos basada en Docker para el desarrollo local. |
+| [Proton](https://github.com/eoscostarica/proton-local)| Red Proton basada en Docker para el desarrollo local. | 
+| [Wax](https://github.com/eoscostarica/wax-local) | WAX Entorno de desarrollo local Docker. | 
+| [EOSIO Explorer](https://github.com/eoscostarica/eosio-explorer)| Una aplicación que proporciona Web GUI para comunicarse con EOSIO blockchain en un entorno de desarrollo local. | 
```

Comment:
Just as an observation, you decide what is better.

In the `Descripción` column, I don't consider it necessary to add the final point.